app/forms/submissions/trial/step03_content_form.rb

```diff
@@ -1,8 +1,9 @@
 module Submissions
   module Trial
     class Step03ContentForm < PlantTrialForm
-      # NOTE: needed because completely empty step breaks submission
-      property :_i_am_fake
+      property :lines_or_varieties, writeable: false
```

Comment:
I admit I'm not that familiar with reform, so I simply tried to be (too) safe here. But does that mean that we can also remove that constraint from e.g. data_status in step 1?